tests/functional/testplan/exporters/testing/test_pdf.py

```diff
@@ -17,6 +18,8 @@
 from testplan.report.testing import styles
 from testplan.testing.multitest.entries import assertions
 
+export_context = ExportContext()
```

Comment:
make this a fixture, and let every testcase get it's own, or just create it in the testcase